### ISSUE TITLE
[Bug]: Crash on start due to space in path

### ISSUE THREAD

**Issue by Avasam** [vscode-restructuredtext/vscode-restructuredtext#430]
### Expected behavior

The extension to start up

### Actual behavior

Constant crashing (until it stops after 5 times)

```
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
[Error - 7:00:21 PM] Connection to server got closed. Server will not be restarted.
```
![image](https://github.com/vscode-restructuredtext/vscode-restructuredtext/assets/1350584/b69edf8c-6090-4ac9-93d0-755113bbda86)

### Detailed steps

Have a space in the executable path. Open a `.rst` file in VSCode

### Extension information

v189.3.0

### Visual Studio Code information

1.83.1 (user setup)

### Operating system information

Version	10.0.19045 Build 19045

### Python information

???

### Sphinx information

???

### Contents of conf.py

```python
N/A
```

### Relevant log output

```shell
[Log - 7:00:21 PM] Server start command: C:\Program Files\Python39\python.exe -m esbonio
[Log - 7:00:21 PM] LanguageClientOptions: {"documentSelector":[{"scheme":"file","language":"restructuredtext"},{"scheme":"file","language":"python"}],"initializationOptions":{"sphinx":{"srcDir":null,"confDir":"","forceFullBuild":true,"numJobs":1,"buildDir":null},"server":{"logLevel":"error","logFilter":[],"hideSphinxOutput":false,"enableScrollSync":true}},"outputChannel":{"name":"Esbonio Language Server"}}
[Log - 7:00:21 PM] Starting Language Server
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
[Info  - 7:00:21 PM] Connection to server got closed. Server will restart.
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
[Info  - 7:00:21 PM] Connection to server got closed. Server will restart.
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
[Info  - 7:00:21 PM] Connection to server got closed. Server will restart.
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
[Info  - 7:00:21 PM] Connection to server got closed. Server will restart.
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
[Error - 7:00:21 PM] Connection to server got closed. Server will not be restarted.
```

**Comment by lextm** [comment]
Duplicate to #406

**Comment by Avasam** [comment]
@lextm I just read through that issue now that you linked it. It has no resolution and thread is locked. If the issue was fixed at some point. It is still currently present. Could you re-open that one if this is considered a duplicate instead of a regression?

If not, then what VSCode extension would you recommend instead to preview `.rst` files?

**Comment by lextm** [comment]
Workarounds were presented in plain English, so I am not sure what's being asked further.
